rpi3.mk

```diff
@@ -184,15 +185,41 @@ rpi3-firmware:
 ifeq ("$(wildcard $(ROOT)/out/$(RPI3_FIRMWARE_FILE).$(RPI3_FIRMWARE_FILE_EXT))","")
 	echo "Downloading Raspberry Pi 3 firmware ..."
 	mkdir -p $(ROOT)/out
+	mkdir -p $(RPI3_STOCK_FW_PATH)/boot
 	wget $(RPI3_FIRMWARE_URL)/$(RPI3_FIRMWARE_FILE).$(RPI3_FIRMWARE_FILE_EXT) -O $(ROOT)/out/$(RPI3_FIRMWARE_FILE).$(RPI3_FIRMWARE_FILE_EXT)
-	unzip -a $(ROOT)/out/$(RPI3_FIRMWARE_FILE).$(RPI3_FIRMWARE_FILE_EXT) -d $(ROOT)
-	mv $(ROOT)/firmware-$(RPI3_FIRMWARE_FILE) $(RPI3_STOCK_FW_PATH)
+	unzip -aj $(ROOT)/out/$(RPI3_FIRMWARE_FILE).$(RPI3_FIRMWARE_FILE_EXT) \
+	'$(RPI3_STOCK_ZIP_BOOT)/bootcode.bin' \
```

Comment:
Or even expand it to also include folders instead of a single variable name and then something like this in the clean rule:
```make
 +		rm -f $(ROOT)/rpi3_firmware/boot/*'; \
 +		rmdir $(RPI3_STOCK_FW_PATH_BOOT); \
 +		rmdir $(RPI3_STOCK_FW_PATH); \
```